css/map-view.css

```diff
@@ -0,0 +1,59 @@
+.map-view iframe {
```

Comment:
Duplicate of 
https://github.com/fossasia/summit.fossasia.org/pull/28/files
You have copied everything from my pull request

Reply:
Thanks for the review. In Open Source using existing code is fine, but the author should be referenced. Please do this next time.

Reply:
I'm sorry you feel this way but I just referenced the code from the current existing map-holder class in the camaro.less file. I realized using plain css would be better than less syntax and updated it with a new class map-view. I also added media-query css rule to update view for mobile with the position attribute.